mz_bokeh_package/utilities/current_user.py

```diff
@@ -0,0 +1,87 @@
+import os
+from bokeh.io import curdoc
+
+from mz_bokeh_package.utilities.environment import Environment
+from mz_bokeh_package.utilities.graphql_api import MZGraphQLClient
+
+
+class CurrentUser:
+    """
+    Class with static methods for getting information about the current user
+    """
+
+    _users = {}
+
+    @classmethod
+    def get_user_info(cls, api_key: str | None) -> dict:
+        """Retrieves user information for the current session or API key, caching the information for future use.
+
+        Args:
+            api_key: unique user api key.
+
+        Returns:
+            dictionary of user "id" and "name".
+
+        Raises:
+             a ValueError if neither a session nor an API key is provided.
```

Comment:
This is not exactly correct. First of all, a session is not provided to this function. I think  the correct statement is: "a ValueError if an API key is not provided and the user info is not cached for this session".